.github/workflows/main.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
       - uses: actions/setup-java@v3
         with:
-          java-version: 11
+          java-version: 17
```

Comment:
Good find. Could you please also change the `java-version` in all the other GitHub workflows to 17?